src/main/java/com/global/api/builders/AuthorizationBuilder.java

```diff
@@ -77,6 +77,7 @@
     private HashMap<String, ArrayList<String[]>> supplementaryData;
     private String tagData;
     private String timestamp;
+    private Integer estimatedNumTxn;
```

Comment:
I would prefer this to be spelled out more. Even if it makes the name ridiculously long.